pom.xml

```diff
@@ -83,11 +83,12 @@
     <version.io.github.openfeign.opentracing>0.1.0</version.io.github.openfeign.opentracing>
     <!-- spring-boot-starter-parent is a module of spring-boot-dependencies
         https://github.com/spring-projects/spring-boot/blob/master/spring-boot-starters/spring-boot-starter-parent/pom.xml -->
-    <version.org.springframework.boot>1.5.12.RELEASE</version.org.springframework.boot>
-    <version.org.springframework.cloud-spring-cloud-dependencies>Edgware.SR3</version.org.springframework.cloud-spring-cloud-dependencies>
+    <version.org.springframework.boot>2.0.3.RELEASE</version.org.springframework.boot>
+    <version.org.springframework.cloud-spring-cloud-dependencies>Finchley.RELEASE</version.org.springframework.cloud-spring-cloud-dependencies>
```

Comment:
We could update to 2.0.4 and SR1 https://github.com/spring-projects/spring-cloud/wiki/Spring-Cloud-Finchley-Release-Notes

But it can be done in a separate PR